addons/core/functions/fnc_playerJipHint.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+/*
+ * Author: PabstMirror
+ * Handles player jipping, sends hint on last group
+ */
+
+
+if (isServer) then {
+    GVAR(playerMap) = createHashMap;
+    [QGVAR(playerConnected), {
+        params ["_player", "_profileName"];
+        private _msg = if (_profileName in GVAR(playerMap)) then {
+            (GVAR(playerMap) get _profileName) params ["_lastSide", "_lastGroup"];
+            private _sideText = ["?", "B", "O", "I"] select ([blufor, opfor, independent] find _lastSide);
```

Comment:
civilian handling doesn't work, it will default to `I` since `find == -1`

Reply:
opps, I think there should have been a subtraction here, I'll just swap to a `switch`